VMBackup/main/parameterparser.py

```diff
@@ -111,10 +112,14 @@ def __init__(self, protected_settings, public_settings, backup_logger):
                 backup_logger.log("settings received " + str(self.dynamicConfigsFromCRP), True)
                 for config in self.dynamicConfigsFromCRP:
                     if CommonVariables.key in config and CommonVariables.value in config:
-                        if(config[CommonVariables.key] in self.wellKnownSettingFlags):
-                            self.wellKnownSettingFlags[config[CommonVariables.key]] = config[CommonVariables.value]
-                        else:
-                            backup_logger.log("The received " + str(config[CommonVariables.key]) + " is not an expected setting name.", True)
+                        found = False
```

Comment:
This loop is not required anymore as we can use the settingKeyMapping dictionary for direct lookup